### PR TITLE
CLN: spelling fixes in docstrings

### PR DESCRIPTION
**Pull request by akosfurton**
- [ ] closes #xxxx
- [x] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [x] whatsnew entry

### CODE REVIEW

File: doc/source/whatsnew/v1.1.0.rst [hunk 1]
```diff
@@ -778,6 +778,7 @@ Other
 - Bug in :meth:`DataFrame.__dir__` caused a segfault when using unicode surrogates in a column name (:issue:`25509`)
 - Bug in :meth:`DataFrame.plot.scatter` caused an error when plotting variable marker sizes (:issue:`32904`)
 - :class:`IntegerArray` now implements the ``sum`` operation (:issue:`33172`)
+- Assorted spelling cleanups in docstrings
```
Comment (mroeschke) [inline doc/source/whatsnew/v1.1.0.rst:781]: Don't need an entry for spelling cleanups in docstrings

**Review by mroeschke — COMMENTED** [review]
Looks like there are some doctest failures with your changes

**Comment by akosfurton** [comment]
Appears that `TestHDFStore.test_append_with_data_columns` is failing in 1 of the 5 TravisCI builds. 

I didn't change that function, and it looks to have passed in all other Azure pipeline builds. Is this a known flaky test?

**Comment by simonjayhawkins** [comment]
> Is this a known flaky test?

yes.

**Review by simonjayhawkins — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @akosfurton! The failing Travis build was unrelated test failure

**Pull request merged**